packages/network/src/provider/utils/rpc-mapper/methods/eth_getBlockByHash/eth_getBlockByHash.ts

```
@@ -1,13 +1,14 @@
 import { ThorId } from '@vechain/sdk-core';
 import { RPC_DOCUMENTATION_URL } from '../../../../../utils';
-import { ethGetBlockByNumber } from '../eth_getBlockByNumber';
 import {
     JSONRPCInternalError,
     JSONRPCInvalidParams,
     stringifyData
 } from '@vechain/sdk-errors';
-import { type BlocksRPC } from '../../../formatter';
+import { blocksFormatter, type BlocksRPC } from '../../../formatter';
 import { type ThorClient } from '../../../../../thor-client';
+import { type DefaultBlock, DefaultBlockToRevision } from '../../../const';
```

ESLint

[error] 'DefaultBlock' is defined but never used.

(@typescript-eslint/no-unused-vars)